data_steward/deid/config/internal_tables/src_hpos_to_allowed_states.csv

```diff
@@ -32,6 +32,7 @@ PIIState_AL,1585261,saou_uab
 PIIState_AL,1585261,saou_uab_hunt
 PIIState_AL,1585261,saou_uab_selma
 PIIState_AL,1585261,saou_umc
+PIIState_AL,1585261,saou_cgmhs
```

Comment:
nit: I'd suggest keeping this list in alphabetical order. It will help pin down issues if we are visually checking

Reply:
I hadn't noticed they were alphabetized by hpo_id.  Thanks for bringing it to my attention.  I updated it.

Reply:
Thank you for updating